### PR TITLE
"auth failed" message implementation

### PR DESCRIPTION
**Pull request by unam3**
~ https://github.com/ForNeVeR/kaiwa/issues/52
check it out.

### CODE REVIEW

**Comment by ForNeVeR** [comment]
Thanks, I'll take a look in the nearest two days (i.e. tonight or tommorow).

File: src/js/helpers/xmppEventHandlers.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ module.exports = function (client, app) {
 
     client.on('auth:failed', function () {
         log.warn('auth failed');
+        localStorage.auth_failed = 1;
```
Comment (ForNeVeR) [inline src/js/helpers/xmppEventHandlers.js]: I'd use `true` here.

File: src/resources/js/login.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
+if (localStorage.auth_failed) {
+    document.getElementById('auth-failed').style.display = 'block';
+    delete localStorage.auth_failed;
```
Comment (ForNeVeR) [inline src/resources/js/login.js]: It seems like it's not the standard-compliant way of deleting things. Better use [`removeItem()`](https://developer.mozilla.org/en/docs/Web/API/Storage/removeItem).

File: src/stylus/pages/login.styl [hunk 3]
```diff
@@ -1,3 +1,8 @@
+#auth-failed
+  display: none;
+  color: red;
```
Comment (ForNeVeR) [inline src/stylus/pages/login.styl]: We're not using color names in the project; please replace with `#f00`.

File: src/js/helpers/xmppEventHandlers.js (hunk 1 above)
Comment (ForNeVeR) [inline src/js/helpers/xmppEventHandlers.js]: Also, use standard snakeCase: `auth_failed` → `authFailed`.

**Comment by unam3** [comment]
doen

**Comment by ForNeVeR** [comment]
Thank you!

**Pull request merged**